stl/inc/format

```diff
@@ -385,15 +386,106 @@ _NODISCARD constexpr const _CharT* _Parse_arg_id(
     throw format_error("Invalid format string.");
 }
 
+inline int _Code_units_in_next_character(const char* _First, const char* _Last, const _Cvtvec& _Cvt) {
+    // Returns a count of the number of code units (characters) that compose the first multibyte

```

Comment:
I suspect that "(characters)" adds more confusion than clarity. Perhaps "(bytes)"?

Reply:
I thought this needed some clarification when I split what was a function template into these two overloads ("Why use the generic term "code units" when they're always bytes?") but looking at it now I think the parentheticals are unnecessary. I _do_ think I should consistently use the term "encoded character" instead of sometimes using "multibyte character".